build-docker.sh

```diff
@@ -21,7 +21,7 @@ esac
 build-silo.sh
 
 ${SUDO} docker build . -t silo:$VERSION
```

Comment:
You'll need to call `docker build` with build arg to specify the base image here. This is must be passed on from whatever environment is provided from the build system (Gitlab / Travis / etc.).